src/Profile.js

```diff
@@ -49,12 +54,18 @@ const Box = styled.div`
     height: 0;
     border: 8px solid black;
     top: -8px;
-    right: 20px;
+    /* width / 3 = width of img */
+    /* width / 3 - 16px (self-width) - 2px (margin) */
```

Comment:
del?

Reply:
Oh these are comments on purpose to explain where this came from